chocs/cookies.py

```diff
@@ -12,4 +12,24 @@ def __init__(
         domain: Optional[str] = None,
         expires: Optional[datetime] = None,
     ):
-        pass
+        self.name = name
+        self.value = value
+        self.path = path
+        self.domain = domain
+        self.expires = expires
+
+        cookie = cookies.SimpleCookie()
+        cookie[name] = str(value)
+        if domain:
+            cookie[name]["domain"] = domain
+        if path:
+            cookie[name]["path"] = path
+        if expires:
+            cookie[name]["expires"] = expires.strftime("%a, %d %b %Y %H:%M:%S GMT")
+        self.cookie = cookie
+
+    def __str__(self) -> str:
+        return str(self.cookie)
+
+    def header(self) -> list:
```

Comment:
Maybe we can name method as following so it is not mistaken with attribute:
`as_header`. Also return type can be hinted a bit better: `List[str, str]`

Reply:
This is funny because I tried that and had problems. Now I notice that you did the same thing as me, it should be `List[str]` (spoiler; list is not a tuple!)

Reply:
Ah yes, you are absolutely right!